### PR TITLE
Bumped version to 0.15.4

### PR DESCRIPTION
**Pull request by archie426**
I have bumped protocal and MCPE version

### CODE REVIEW

**Comment by SOF3** [comment]
Just look and you'll probably find many duplicates.

**Comment by archie426** [comment]
Oops. Soz. Althoguh none have bben accepted though.
Why were u banned?

**Comment by dktapps** [comment]
None have been accepted because nobody is maintaining this repository. Why should yours be any different?

(I'd also recommend that you refrain from off-topic chatter.)

File: src/pocketmine/network/protocol/Info.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	/**
 	 * Actual Minecraft: PE protocol version
 	 */
-	const CURRENT_PROTOCOL = 81;
+	const CURRENT_PROTOCOL = 82
```
Comment (Leo3418) [inline src/pocketmine/network/protocol/Info.php]: Oops.

Comment (PrimusLV) [inline src/pocketmine/network/protocol/Info.php]: *where's the facepalm emoji*

Comment (SOF3) [inline src/pocketmine/network/protocol/Info.php]: @PrimusLV avoid using PR comments as a forum.

Comment (archie426) [inline src/pocketmine/network/protocol/Info.php]: I feel so stupid, xD

**Pull request closed without merging**